proto/recon.proto

```diff
@@ -126,7 +126,12 @@ message BulkFindEntitiesResponse {
   repeated Entity entities = 1;
 }
 
-message RecognizePlacesRequest { repeated string queries = 1; }
+message RecognizePlacesRequest {
+  repeated string queries = 1;
+  // When set, bogus name should be resolved.
+  // This needs to be set for ResoveDescription, but not set for RecognizePlaces for sentences.
+  bool resolve_bogus_name = 2;
```

Comment:
nit: maybe call it `require_context_for_common_names`?  we internally call it bogus name.

Reply:
Per offline discussion - making this an internal flag.